tests/organizations_member_connection.spec.js

```diff
@@ -0,0 +1,129 @@
+const axios = require('axios');
+const { URL } = require('../constants');
+const getToken = require('./functions/getToken');
+
+let token;
+
+beforeAll(async () => {
+  token = await getToken();
+});
+
+describe('organization member connection resolvers', () => {
+  let createdOrgId;
+
+  test('createOrganization', async () => {
+    const createdOrgResponse = await axios.post(
+      URL,
+      {
+        query: `
+            mutation {
+                createOrganization(data: {
+                    name:"test org"
+                    description:"test description"
+                    isPublic: true
+                    visibleInSearch: true
+                    }) {
+                        _id
+                    }
+            }
+              `,
+      },
+      {
+        headers: {
+          Authorization: `Bearer ${token}`,
+        },
+      }
+    );
+    const { data } = createdOrgResponse;
+    createdOrgId = data?.data.createOrganization._id;
+    expect(data.data.createOrganization).toEqual(
+      expect.objectContaining({
+        _id: expect.any(String),
+      })
+    );
+  });
+
+  test('Organization Member Connection', async () => {
```

Comment:
This is one case. You need to add other cases as well
1. A user provides the first parameter as input but doesn't provide a skip then the error "Missing Skip parameter. Set it to either 0 or some other value" should appear.
2. Create a request without token and error should appear accordingly

Reply:
When a query request is sent without an Auth Token, the output is the same as that of a query request with the token.
So, I didn't add any test case for it. Is that Okay?

Reply:
Go to `talawa-api/schema/query.graphql` and add @auth there and restrict the API access for the verified user only. Then add test code. This is important

Reply:
> Please add a snapshot of all these test code results in the current repo. I want to check if they are passing.

Definetely.

![image](https://user-images.githubusercontent.com/57038543/146671229-d2e780f5-0ebc-462a-9e72-8bd6f0e27bc7.png)